core/src/main/scala/cats/data/NonEmptyVector.scala

```diff
@@ -216,6 +216,9 @@ private[data] sealed trait NonEmptyVectorInstances {
       override def foldRight[A, B](fa: NonEmptyVector[A], lb: Eval[B])(f: (A, Eval[B]) => Eval[B]): Eval[B] =
         fa.foldRight(lb)(f)
 
+      override def get[A](fa: NonEmptyVector[A])(idx: Long): Option[A] =
+        fa.get(idx.toInt)
```

Comment:
I don't think it makes sense to offer size as a long and get with an `idx: Int`. :+1: to checking if it fits if an `Int` and returning `None` otherwise.